src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterOptions.cs

```diff
@@ -0,0 +1,121 @@
+// <copyright file="OtlpMetricExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Metrics;
+
+namespace OpenTelemetry.Exporter;
+
+/// <summary>
+/// OpenTelemetry Protocol (OTLP) exporter options for metric specific OTLP exporter configuration.
+/// OTEL_EXPORTER_OTLP_METRICS_ENDPOINT, OTEL_EXPORTER_OTLP_METRICS_HEADERS, OTEL_EXPORTER_OTLP_METRICS_TIMEOUT, OTEL_EXPORTER_OTLP_METRICS_PROTOCOL
+/// environment variables are parsed during object construction.
+/// </summary>
+/// <remarks>
+/// The constructor throws <see cref="FormatException"/> if it fails to parse
+/// any of the supported environment variables.
+/// </remarks>
+public class OtlpMetricExporterOptions : ICommonOtlpExporterOptions
```

Comment:
One potential thing to consider with this design - if later there is a single OTLP exporter instance (which shares the same TCP connection across metrics and traces), what `Options` type will it use?

Reply:
My hope is that the design will not prevent us from doing this. Folks would still use the `OtlpExporterOptions` class. I intentionally made the constructor of `OtlpMetricExporterOptions` internal - the only way to get a handle on it is through an instance of `OtlpExporterOptions`.

Reply:
So I think it might look something like:

```csharp
class MegaOtlpExporter
{
    private TraceOltpExporter traceExporter;
    private MetricOtlpExporter metricExporter;

    public MegaOtlpExporter(OtlpExporterOptions options)
    {
        // Figure out from options if the Endpoint, Headers, etc are all the same for traces and metrics
        // If so, then construct the trace and metric exporters with the same channel.
        // Otherwise, we gotta construct multiple channels.
    }
}
```

Reply:
+1 on the name "mega" 😆